pkgs/development/tools/build-managers/sbt/default.nix

```diff
@@ -2,15 +2,15 @@
 
 stdenv.mkDerivation rec {
   pname = "sbt";
-  version = "1.2.8";
+  version = "1.3.2";
 
   src = fetchurl {
     urls = [
       "https://dl.bintray.com/sbt/native-packages/sbt/${version}/${pname}-${version}.tgz"
       "https://github.com/sbt/sbt/releases/download/v${version}/sbt-${version}.tgz"
       "https://cocl.us/sbt-${version}.tgz"
     ];
-    sha256 = "0n7yghnb1q3lyjpv721znvslk5lwib7y84mxwz66yv8p84jj3fcv";
+    sha256 = "1ncjc4yrnhp2kc5xw3ismnyx5n71541760rzs81hxi8x8vi36x5b";
```

Comment:
I get a different hash:

```
hash mismatch in fixed-output derivation '/nix/store/v8f3lbkzkr5gjkm6m794xhzrqfnnpmiz-sbt-1.3.2.tgz':
  wanted: sha256:1ncjc4yrnhp2kc5xw3ismnyx5n71541760rzs81hxi8x8vi36x5b
  got:    sha256:0vrj9wlw5kf6w7nzhf58dyik0n7rnc9aivjpsz85m299j4wyz37d
```

Reply:
@Mic92 interesting ... not sure if this is the correct way to get the hash:
```
nix-prefetch-url --unpack https://github.com/sbt/sbt/releases/download/v1.3.2/sbt-1.3.2.tgz
unpacking...
[49.4 MiB DL]
path is '/nix/store/2mvik6xvnas8s3i8c78lmlljq15a2g3l-sbt-1.3.2.tgz'
1ncjc4yrnhp2kc5xw3ismnyx5n71541760rzs81hxi8x8vi36x5b
```

Reply:
@regadas is it possible that the bintray archive is different from the github one?

Reply:
Hi @Mic92 it's definitely possible; tbh only the github link is being updated with the latest releases the others will just 404; I think we should just keep the github one, wdyt?

Reply:
@regadas yes. drop the other ones.

Reply:
updated ... and they have the same hash using `nix-prefetch-url`

Reply:
The checksum is still incorrect.

Reply:
@Mic92 hmm,

```
nix-prefetch-url --unpack https://piccolo.link/sbt-1.3.2.tgz    unpacking...
path is '/nix/store/2mvik6xvnas8s3i8c78lmlljq15a2g3l-sbt-1.3.2.tgz'
1ncjc4yrnhp2kc5xw3ismnyx5n71541760rzs81hxi8x8vi36x5b

nix-prefetch-url --unpack https://github.com/sbt/sbt/releases/download/v1.3.2/sbt-1.3.2.tgz      
unpacking...
path is '/nix/store/2mvik6xvnas8s3i8c78lmlljq15a2g3l-sbt-1.3.2.tgz'
1ncjc4yrnhp2kc5xw3ismnyx5n71541760rzs81hxi8x8vi36x5b
```

Reply:
I'll have a better look at this later ... looks like smth else might be broken?

Reply:
Ofborg also shows a different checksum: https://logs.nix.ci/?key=nixos/nixpkgs.69327&attempt_id=91c42493-27e4-4271-8664-3e6aa9c037fd

Can you build the package manually?

Reply:
Ah. You should not use `--unpack` when `fetchurl` is used. Fetchurl is calculating the checksum of the tarball and not of the directory content.

Reply:
Ahh ok! i thought it would check the content ... good to know 👍